8.1.2/startup/initAppInsights.js

```diff
@@ -0,0 +1,8 @@
+try {
+    // Initialize the Application Insights runtime, using the
+    // instrumentation key provided by the environment.
+    require("applicationinsights").setup().start();
```

Comment:
@nickwalkmsft So I confirmed that all AI SDK config can be overridable in app code (e.g. enabling certain types of auto-collected telemetry), *except* for the actual instrumentation key. However, the app could work around this by simply setting the `APPINSIGHTS_INSTRUMENTATIONKEY` environment variable (which is actually the recommendation for production anyways), and then everything would work as expected, since the env var would naturally be used by the "first" initialization of the AI SDK.